lib/recipes/organic-chemistry/locales/en.yml

```diff
@@ -1,11 +1,12 @@
 en:
   example: Worked Example
+  section_exercises: ! '%{number} Problems'
```

Comment:
Do we need the %{number} there? the spec does just say "Problems"

Reply:
It's a new change, discussed here: https://github.com/openstax/cnx-recipes/issues/4749#issuecomment-1197030961